apple/macbook-pro/11-5/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, config, pkgs, ... }:
+
+let
+
+  kernelPackages = config.boot.kernelPackages;
+
+in
+
+{
+  imports = [
+    ../.
+    ../../../common/pc/laptop/ssd
+    <nixpkgs/nixos/modules/hardware/network/broadcom-43xx.nix>
+  ];
+
+  boot.loader.systemd-boot.enable = true;
```

Comment:
I think mkDefault should be used for boolean options: https://github.com/NixOS/nixos-hardware/pull/67/files#diff-0660552a9aaaac3267076efad7bf5c1bR9

Reply:
Fair enough, seems reasonable to make this optional.